package ch9.exception;

public class PracticeExceptionMain {
    public static void main(String[] args) {
        try{
            example();
        }catch(Exception e){
            System.out.println("예외 발생");
        }

    }
    private static void example() throws Exception{

    }
}
